app/ide-desktop/lib/file-watcher/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "enso-file-watcher",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "typecheck": "tsc --noEmit",
+    "lint": "npx --yes eslint src",
+    "build": "tsx bundle.ts",
+    "start": "tsx bundle.ts && node dist/index.cjs"
+  },
+  "dependencies": {
+    "chokidar": "^3.5.3",
+    "yargs": "^17.7.2"
+  },
+  "devDependencies": {
+    "@exact-realty/esbuild-plugin-inline-js": "^1.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
+    "@typescript-eslint/parser": "^5.49.0",
+    "eslint": "^8.32.0",
+    "eslint-plugin-jsdoc": "^39.6.8",
+    "typescript": "^4.9.4"
+  },
+  "optionalDependencies": {
+    "@esbuild/darwin-x64": "^0.17.15",
+    "@esbuild/linux-x64": "^0.17.15",
+    "@esbuild/windows-x64": "^0.17.15"
+  }
+}
```

Comment:
no nl

Reply:
will revert it, but this nl is added by ./run fmt